trillian_map_api.proto

```diff
@@ -58,7 +59,15 @@ message GetMapLeavesResponse {
 message SetMapLeavesRequest {
   int64 map_id = 1;
   repeated MapLeaf leaves = 2;
-  MapperMetadata mapper_data = 3;
+  reserved 3;  // was MapperMetadata (removed, replaced by mapper_metadata).
+  // Metadata that the Map should associate with the new Map root after
+  // incorporating the changes in leaves.  The metadata will be reflected in
+  // the Map Root returned in SetMapLeavesResponse.
+  // Map personalities should use metadata to persist any state needed later
+  // to continue mapping from an external data source. Any protocol message
+  // can be assigned, via Any marshalling (see github.com/golang/protobuf/ptypes).
+  // Size limitations may apply dependent on the Trillian storage type in use.
```

Comment:
I think it's OK to not talk about the size limit at this layer if you prefer. We don't talk specifically about size limits in other places that do have them, plus I think it's fine to omit this error as long as we have good error messages.

Reply:
OK, done.